### PR TITLE
Removed duplicate license files (fixes #3)

### LINKED ISSUE #3: reduce duplication of LICENSE
Currently, the `LICENSE` file is stored in three places:

* repo root
* `py-pkg/LICENSE`
* `r-pkg/LICENSE`

This duplication is unnecessary and could lead to inconsistencies in the future. To close this issue, set up a shell script of make file or something which copies the root `LICENSE` into the packages. Something like

_Makefile_

```
build-r:
    cp LICENSE r-pkg/
    R CMD build r-pkg/

build-py:
    cp LICENSE py-pkg/
    cd py-pkg && python3 setup.py sdist
```

### PR DESCRIPTION
**Pull request by chen3e**

### CODE REVIEW

**Review by jameslamb — CHANGES_REQUESTED** [review]
Thanks for the contribution! This still needs some work

1. Could you please add `(fixes #3)` to the title of the PR? This will automatically close the issue once this PR is merged
2. Can you please remove the `all` recipe? It's not necessary at this point
3. Can you please remove the `LICENSE` files that currently exist in the `r-client` and `py-client` . directories?
4. Can you please remove the excessive space before the commands in each recipe? One tab should be sufficient.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+build_r:
+	cp LICENSE clients/r-client
+	R CMD build clients/r-client
+
+build_py:
+	cp LICENSE clients/py-client
+	cd py-pkg && python3 setup.py sdist
```
Comment (jameslamb) [inline Makefile]: can you please add a new line at the end of this file?

**Review by jameslamb — CHANGES_REQUESTED** [review]
Thanks for the changes! Looks good to me. Could you please change the title of the PR? It should have a human-readable title so we can find it again in the future through search.

I recommend `Removed duplicate license files (fixes #3)`

File: Makefile (hunk 1 above)
Comment (chen3e) [reply Makefile]: Sure thing- sorry about that.

Comment (jameslamb) [inline Makefile]: ok last thing...`py-pkg/` is not a directory in this project. Could you change this to `cd . clients/py-client`?

**Review by jameslamb — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (chen3e) [reply Makefile]: Done- can't believe I missed that one!

**Review by jameslamb — APPROVED** [review]
looks good, thanks for the contribution!

**Pull request merged**